Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -472,7 +472,8 @@ - (void)_setupPageConfiguration:(Ref<API::PageConfiguration>&)pageConfiguration
     pageConfiguration->preferences()->setShouldRespectImageOrientation(!![_configuration _respectsImageOrientation]);
 #if !PLATFORM(MAC)
     // FIXME: Expose WKPreferences._shouldPrintBackgrounds on iOS, adopt it in all iOS clients, and remove this and WKWebViewConfiguration._printsBackgrounds.
```

Comment:
This comment should be updated to reflect the fact that we have API now.

Can leave a FIXME with a new radar to track the SPI removal on iOS.